modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/handlers/fieldSetAddedHandler.es.js

```diff
@@ -33,14 +32,7 @@ const handleFieldSetAdded = (props, state, event) => {
 	const nestedFields = [];
 
 	visitor.mapFields((nestedField) => {
```

Comment:
This all commit changes belongs to use a fieldSet story, and for this reason will no longer be used by this PR!